tests/samples/passwd.groovy

```diff
@@ -1 +1,3 @@
 gi_reo_gi_passwd = "cAc48k1Zd7"
+mypw: KrAcMe12345,
+masked_password = *****1*****2;
```

Comment:
The filter which is for the masked case will be added on next PR?

Reply:
Actually, the filter presents https://github.com/Samsung/CredSweeper/blob/2b3e47de2f505fa09641d18e78266c8d96988a62/credsweeper/filters/value_allowlist_check.py#L15
The  sample is like a "canary".

Reply:
Oh I see.

Reply:
As I tested it from my PC it seems it doesn't work.. Could you please check this?
``` bash
$ python -m credsweeper --path tests/samples/passwd.groovy --doc
rule: Password / severity: medium / line_data_list: [line: 'gi_reo_gi_passwd = "cAc48k1Zd7"' / line_num: 1 / path: tests/samples/passwd.groovy / value: 'cAc48k1Zd7' / entropy_validation: BASE64_CHARS 3.121928 False] / api_validation: NOT_AVAILABLE / ml_validation: VALIDATED_KEY
rule: Password / severity: medium / line_data_list: [line: 'mypw: KrAcMe12345,' / line_num: 2 / path: tests/samples/passwd.groovy / value: 'KrAcMe12345,' / entropy_validation: BASE64_CHARS 3.286216 False] / api_validation: NOT_AVAILABLE / ml_validation: VALIDATED_KEY
Detected Credentials: 2
Time Elapsed: 0.09668970108032227s
```

I thought `maksed_password` should be detected with `--doc` option.. isn't it?

Reply:
Probably the pattern "*****" should be moved to the excluded filter.